<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inserimentoturno_window</class>
 <widget class="QMainWindow" name="inserimentoturno_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>492</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>792</width>
    <height>492</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>792</width>
    <height>492</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modifica Turno</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xz.qrc">
    <normaloff>:/images/images/AnconAmbiente_Logo.png</normaloff>:/images/images/AnconAmbiente_Logo.png</iconset>
  </property>
  <widget class="QWidget" name="inserimentoturno_widget">
   <widget class="QTabWidget" name="inserimentoturno_tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>421</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>771</width>
      <height>421</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>771</width>
      <height>421</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="dati_tab">
     <attribute name="title">
      <string>Dati</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Servizi da inserire:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabella_servizi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>751</width>
        <height>191</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tipo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Luogo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Periodicit√†</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="servizio_error">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>350</y>
        <width>411</width>
        <height>16</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>190</red>
            <green>190</green>
            <blue>190</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>190</red>
            <green>190</green>
            <blue>190</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>343</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Selezionare ID Servizio:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_servizi">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>340</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Data e ora fine turno:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>23</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Data e ora inizio turno:</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="inizio_datetimepicker">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>194</width>
        <height>26</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>M/d/yy hh:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="fine_datetimepicker">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>67</y>
        <width>194</width>
        <height>26</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>M/d/yy hh:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="servizi_button">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>110</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
      </property>
      <property name="text">
       <string>LISTA COMPLETA SERVIZI</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mezzi_widget">
     <attribute name="title">
      <string>Mezzi</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleziona i mezzi dalla tabella</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabella_mezzi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>751</width>
        <height>301</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Modello</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Allestimento</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stato</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="warning_label2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>353</y>
        <width>551</width>
        <height>16</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>190</red>
            <green>190</green>
            <blue>190</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>190</red>
            <green>190</green>
            <blue>190</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="operatori_tab">
     <attribute name="title">
      <string>Operatori</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleziona gli operatori dalla tabella</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabella_operatori">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>751</width>
        <height>301</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>170</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cognome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stato</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="operatori_error">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>354</y>
        <width>551</width>
        <height>16</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>190</red>
            <green>190</green>
            <blue>190</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>190</red>
            <green>190</green>
            <blue>190</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="salva_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>438</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>SALVA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="annulla_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>438</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(24, 105, 68);
color: white;
border-style: outset;
font:  bold 11px


}

QPushButton:hover{
background-color:rgb(170, 203, 108);
color: white;
border-style: outset;
font:  bold 11px
}</string>
    </property>
    <property name="text">
     <string>ANNULLA</string>
    </property>
   </widget>
   <widget class="QLabel" name="inserimento_error">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>444</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="xz.qrc"/>
 </resources>
 <connections/>
</ui>
